src/iter/internal.rs

```diff
@@ -133,7 +133,14 @@ pub trait UnindexedProducer: Send + Sized {
 /// job is actually stolen into a different thread.
 #[derive(Clone, Copy)]
 struct Splitter {
+
```

Comment:
Nah, that's my doing.  I go back and forth on preferring a blank line before doc comments, but I'll remove it if you find this nit-worthy. :)